app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="placeholder_session_title">ハードウェア攻撃、高度なARM悪用、およびAndroidハッキング</string>
+    <string name="placeholder_session_subtitle">最近研究の相談、技術のライブデモ</string>
+    <string name="placeholder_session_track">セキュリティーと安全性</string>
+    <string name="placeholder_session_abstract">一部がInfiltrate2013とNoSuchCon2013で配信されたこの講演では、（組み込みアプリケーションと携帯端末向け）LinuxとAndroidでの（今年と最後にBlackhatで完売した）Practical ARM Exploitationコースに組み込まれている最近の研究について説明します。 また、これらの技術をデモし、カスタム構築したいくつかのARMハードウェア開発プラットフォームを使用して、それらを発見する方法について説明します。 必要に応じて、共著で2013年10月にリリースされるAndroid Hackers Handbookで説明したように、Androidに関連するARMの悪用についても説明します。</string>
+    <string name="placeholder_session_description">最後に、バスプロトコルの盗聴/リバースエンジニアリング、ハードウェアデバッグを明らかにし、ハードウェア技術を使用して組み込みソフトウェア（ファームウェア）を密かに入手するなど、（上記を容易にするため）最新の関連ハードウェア研究についても説明します。この作業を実行し、Apple MFI iAPデバイス、ならびOEM実装のUSBスタックを使用するマルチメディアデバイスに対してそれらをデプロイするために、使用されるサポートツールと開発された技術を示し、デモします。 （これには、ハードウェアUSBファザーの完全に組み立てられた修正バージョンを販売するhttp://int3.ccの開始に関する経験が簡単に含まれます。）   途中に、（ゼロから）ハードウェアを完全に設計し、ファームウェアを書き、特許を取得した研究者向けのハードウェア脆弱性評価スイスアーミーナイフとして公開しているOspreyと呼ばれる組み込みセキュリティデバイス用のモバイルアプリで学んだ教訓の一部を必然的に共有します。</string>
+    <string name="placeholder_session_speakers">Stephen A. Ridley, Ernest Ridley, Conan Ridley, Bridget Ridley, Frank Ridley</string>
+    <string name="placeholder_session_links">Don\'t Stuff Beans Up Your Nose INT3.CC The Osprey (on INT3)</string>
+    <string name="placeholder_session_online">https://events.ccc.de/congress/2013/Fahrplan/events/5193.html</string>
+    <string name="placeholder_session_language">ja</string>
+    <string name="placeholder_session_day">今日</string>
+    <string name="placeholder_session_start_time">17:15</string>
+    <string name="placeholder_session_duration">240分</string>
+    <string name="placeholder_session_room">Village:Königlich Bayerisches Amtsvillage</string>
+    <string name="placeholder_schedule_time">09:15</string>
+    <string name="placeholder_alarm_time">2013年12月28日　午後5時10分</string>
+    <string name="placeholder_session_date">2013年12月28日　午後5時15分</string>
+    <string name="placeholder_session_location">本当に広い部屋</string>
+    <string name="placeholder_session_id">ID: 5193</string>
+    <string name="placeholder_day_separator_title">2日目</string>
```

Comment:
Thank you for the effort but please don't translate the `placeholder_` strings as pointed out in the issue description. These strings are just used for development purposes and do not show up in the app. Please remove them from this file.

Reply:
Thanks for your kindly detail comments.
I've fixed them all, and the build also passed. Please check!
- Rebased into 1 commit, updated with push -force
- Deleted preference and placeholder translation in `strings.xml`
- Removed " " in `strings.xml`

Reply:
Thanks for the update. A few more corrections and then we will be good.